src/group_theory/coset.lean

```diff
@@ -3,8 +3,37 @@ Copyright (c) 2018 Mitchell Rowett. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mitchell Rowett, Scott Morrison
 -/
+
 import group_theory.subgroup
 
+/-!
+# Cosets
+
+This file develops the basic theory of left and right cosets.
+
+## Main definitions
+
+* `left_coset a s`: the left coset `a * s` corresponding to an element `a : α` and a subset `s ⊆ α`.
```

Comment:
```suggestion
* `left_coset a s`: the left coset `a * s` for an element `a : α` and a subset `s ⊆ α`.
```
Not too sure about the phrasing here. (Same for the next few "corresponding to")

Reply:
I replaced some "corresponding to". Let me know what you think.